migrations/20200804025157-create-models.js

```diff
@@ -0,0 +1,308 @@
+"use strict";
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+    await queryInterface.sequelize.query(`
+      -- ER/Studio Data Architect SQL Code Generation
+      -- Project :      BATS Logical Data Model.DM1
+      --
+      -- Date Created : Sunday, August 02, 2020 18:13:09
+      -- Target DBMS : PostgreSQL 9.x
+      --
+      -- 
+      -- TABLE: ambulance 
+      --
+      CREATE TABLE ambulance(
+          ambulance_uuid                          uuid           NOT NULL,
+          emergencymedicalserviceprovider_uuid    uuid           NOT NULL,
+          ambulanceidentifier                     varchar(50)    NOT NULL,
+          recordcreatetimestamp                   timestamp      NOT NULL,
+          recordcreatesource                      varchar(50)    NOT NULL,
```

Comment:
@billhoke what are you expecting to store in the "source" columns for each record?

If we create user accounts, these could potentially change to be foreign key references to the accounts that were logged in at the time of record creation.

Reply:
@francisli because we don't yet know if (or what) authentication will be required for specific users, especially in the hospitals, I made this deliberately vague and defined it as text so that it could be adapted as we learn more, possibly without change to the structure. Of course, if we end up with our own user table and internal authentication framework, we will probably change it to a uuid foreign key back to a user table. All TBD, it seems ... and hoping Lissa, Callum and I can get some meetings setup soon with the hospitals to determine the minimum related requirements. Does that sound okay? Any thoughts from your side?

Reply:
@billhoke sounds fine, was just wondering if you had something specific in mind at that time